template.rb

```diff
@@ -89,6 +89,9 @@ def copy_solidus_starter_frontend_files
   copy_file 'config/initializers/solidus_auth_devise_unauthorized_redirect.rb'
   copy_file 'config/initializers/canonical_rails.rb'
 
+  copy_file 'config/initializers/devise.rb', 'tmp/devise.rb'
+  append_file 'config/initializers/devise.rb', File.read('tmp/devise.rb')
```

Comment:
nit, probably for another PR: I think we could have used https://github.com/rails/thor/blob/v1.2.1/lib/thor/actions/inject_into_file.rb#L26-L33 here as it supports injecting into existing blocks